### PR TITLE
Issue #329: New schema to support TV listings.

### PR DESCRIPTION
**Pull request by vholland**

### CODE REVIEW

File: data/schema.rdfa [hunk 1]
```diff
@@ -11328,6 +11329,64 @@ postponing for 1.6.
...
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Intangible">Intangible</a></span>
+   </div>
+    <div typeof="rdf:Property" resource="http://schema.org/televisionChannelId">
+      <span class="h" property="rdfs:label">televisionChannelId</span>
+      <span property="rdfs:comment">The unique address by which the TV station can be identified in a provider lineup. In US, this is typically a number.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/TelevisionChannel">TelevisionChannel</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/inBroadcastLineup">
+      <span class="h" property="rdfs:label">inLineup</span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Your label still says "inLineup"

Comment (vholland) [inline data/schema.rdfa]: Good catch. I updated the pull request with the new label.


Vicki Tardif Holland | Ontologist | vtardif@google.com

File: data/schema.rdfa [hunk 2]
```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
     <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
     <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/URL">URL</a></span>
 </div>
+
+﻿<div>
+  <h1>TV Listing additions</h1>
+    <div typeof="rdf:Property" resource="http://schema.org/broadcastAffiliateOf">
+      <span class="h" property="rdfs:label">broadcastAffiliateOf</span>
+      <span property="rdfs:comment">The television network(s) whose content is broadcast on this station.</span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Should this be reworded to also include radio?

File: data/schema.rdfa [hunk 3]
```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
...
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/broadcastDisplayName">
+      <span class="h" property="rdfs:label">broadcastDisplayName</span>
+      <span property="rdfs:comment">The name displayed in the channel guide. For many US affiliates, it is the network name.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastService">BroadcastService</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/timezone">
+      <span class="h" property="rdfs:label">timezone</span>
+      <span property="rdfs:comment">The timezone in &lt;a href=&#39;http://en.wikipedia.org/wiki/ISO_8601&#39;&gt;ISO 8601 format&lt;/a&gt;for which the TV station bases its broadcasts.</span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Another TV-specific wording.

File: data/schema.rdfa [hunk 4]
```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
...
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastService">BroadcastService</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/isLiveBroadcast">
+      <span class="h" property="rdfs:label">isLiveBroadcast</span>
+      <span property="rdfs:comment">True is the broadcast is of a live event..</span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Extra period

File: data/schema.rdfa [hunk 5]
```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
...
+   </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/BroadcastChannel">
+      <span class="h" property="rdfs:label">BroadcastChannel</span>
+      <span property="rdfs:comment">A unique instance of a BroadcastService on a CableOrSatelliteService lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/Intangible">Intangible</a></span>
+   </div>
+    <div typeof="rdfs:Class" resource="http://schema.org/TelevisionChannel">
+      <span class="h" property="rdfs:label">TelevisionChannel</span>
+      <span property="rdfs:comment">A unique instance of a television BroadcastService on a CableOrSatelliteService lineup.</span>
+      <span>Subclass of: <a property="rdfs:subClassOf" href="http://schema.org/BroadcastChannel">Intangible</a></span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Intangible -> BroadcastChannel

File: data/schema.rdfa [hunk 6]
```diff
@@ -11894,6 +11894,82 @@ Standards bodies should promote a standard prefix for the identifiers of propert
...
+    <div typeof="rdf:Property" resource="http://schema.org/broadcastChannelId">
+      <span class="h" property="rdfs:label">broadcastChannelId</span>
+      <span property="rdfs:comment">The unique address by which the BroadcastService can be identified in a provider lineup. In US, this is typically a number.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastChannel">BroadcastChannel</a></span>
+      <span>Range: <a property="http://schema.org/rangeIncludes" href="http://schema.org/Text">Text</a></span>
+    </div>
+    <div typeof="rdf:Property" resource="http://schema.org/inBroadcastLineup">
+      <span class="h" property="rdfs:label">inBroadcastLineup</span>
+      <span property="rdfs:comment">The CableOrSatelliteService offering the channel.</span>
+      <span>Domain: <a property="http://schema.org/domainIncludes" href="http://schema.org/BroadcastService">BroadcastService</a></span>
```
Comment (tmarshbing) [inline data/schema.rdfa]: Should the domain be BroadcastService or BroadcastChannel? The examples have it on BroadcastChannel, which makes more sense to me.

File: data/schema.rdfa (hunk 2 above)
Comment (vholland) [inline data/schema.rdfa]: Fixed.

File: data/schema.rdfa (hunk 3 above)
Comment (vholland) [inline data/schema.rdfa]: Fixed.

File: data/schema.rdfa (hunk 4 above)
Comment (vholland) [inline data/schema.rdfa]: Fixed.

File: data/schema.rdfa (hunk 5 above)
Comment (vholland) [inline data/schema.rdfa]: Fixed (both).

File: data/schema.rdfa (hunk 6 above)
Comment (vholland) [inline data/schema.rdfa]: Fixed.

**Pull request merged**